fs/metrics.h

```diff
@@ -0,0 +1,106 @@
+#pragma once
```

Comment:
New `.h` file should be added to https://github.com/westerndigitalcorporation/zenfs/blob/master/zenfs.mk

Reply:
OK.